Action/src/Action.java

```diff
@@ -0,0 +1,109 @@
+import java.io.BufferedReader;
+import java.io.IOException;
+import java.io.InputStreamReader;
+
+public class Action {
+    public static void main(String[] args) throws IOException {
+        int countOfWater = 400;
+        int countOfMilk = 540;
+        int countOfCoffeeBeans = 120;
+        int countOfMoney = 550;
+        int countOfCups = 9;
+
+        BufferedReader reader = new BufferedReader(new InputStreamReader(System.in));
```

Comment:
@AraChilingaryan do you know what is bufferReader ?

Reply:
yes  i know

Reply:
erb konsolic ban enq grum..erb chogtagorcenq bufferedreadery apa karox enq mer gracy kardal bayt ar bayt..inchy dandaxa...dra hamar bufferizaciaenq anum..inqy karduma vorosh qanakov bufferi mej..u menq arden ashxatum enq dra het voch te miangamic kardum enq..araganuma dranov..patoknerov enq eli kardum